packages/design-system/src/components/Tooltip/TooltipIcon.tsx

```diff
@@ -1,22 +1,34 @@
 import { InfoCircleIconThin } from '../Icons';
+import { IconCommonProps } from '../Icons/SvgIcon';
 import React from 'react';
 import classNames from 'classnames';
 
-export interface TooltipIconProps {
+export interface TooltipIconProps extends IconCommonProps {
+  /**
+   * Optional way to provide an accessible name to the tooltip icon that is not the <title> element.
+   * Use only if the tooltip icon is not contained in an element that has an accessible name.
+   */
+  ariaLabel?: string;
```

Comment:
Is this prop a misnomer? It's not actually used as an `aria-label` attribute anywhere, is it? I don't care that much, but I thought I'd flag it as potentially needing a new name now that the implementation is decided on.